### ISSUE TITLE
Client Connector Error

### ISSUE THREAD

**Issue by garyak** [PeteRager/lennoxs30#180]
**Describe the bug**
Unable to connect to s30.
**To Reproduce**
Migrated from 2022.6.7 to 2022.7.3. 

**Expected behavior**
Integration was unable to connect with the s30 on the latest HA release. Last working release is 2022.6.7

**Screenshots/Logs**
```
2022-07-11 07:00:38 ERROR (MainThread) [custom_components.lennoxs30] async_setup unexpected error login Client Connector Error - failed due host not reachable url [https://192.168.3.94/Endpoints/ha_prod/Connect]
2022-07-11 07:01:38 ERROR (MainThread) [custom_components.lennoxs30] async_setup host [192.168.3.94] unexpected error Code [11] Reference [400] [login Client Connector Error - failed due host not reachable url [https://192.168.3.94/Endpoints/ha_prod/Connect]]
2022-07-11 07:02:38 ERROR (MainThread) [custom_components.lennoxs30] async_setup host [192.168.3.94] unexpected error Code [11] Reference [400] [login Client Connector Error - failed due host not reachable url [https://192.168.3.94/Endpoints/ha_prod/Connect]]
```

**Version**
 - Home Assistant: 2022.7.3
 - This Integration: 2.9
 

**Integration Configuration **
- Cloud or Local Connection: local
- Inverter or Sensor Diagnostics Enabled: Not Enabled

**Additional context**

**Comment by PeteRager** [comment]
This issue should be fixed in 0.2.10

https://github.com/PeteRager/lennoxs30/releases/tag/0.2.10

**Comment by garyak** [comment]
Update to 2.10 corrected the error.
